allennlp/modules/__init__.py

```diff
@@ -21,3 +21,7 @@
 from allennlp.modules.text_field_embedders import TextFieldEmbedder
 from allennlp.modules.time_distributed import TimeDistributed
 from allennlp.modules.token_embedders import TokenEmbedder, Embedding
+from allennlp.modules.dot_product_matrix_attention import DotProductMatrixAttention
```

Comment:
Oh, one other note that I forgot last time - it'd be nice to make a `matrix_attention/` module, and put all of these in there, like how the `seq2seq_encoders` are structured.